package.json

```diff
@@ -268,7 +268,7 @@
     "resin-cli-visuals": "^1.8.0",
     "resin-compose-parse": "^2.1.3",
     "resin-doodles": "^0.1.1",
-    "resin-multibuild": "^4.12.1",
+    "resin-multibuild": "4.12.2-logger-for-multiarch-warnings-277a26c5e7ea11c2c4bba007263f4c168232682d",
```

Comment:
I've taken the liberty of adding a commit to this PR and editing the commit message of the previous commit.

This temporary `resin-multibuild` version refers to a draft multibuild PR: https://github.com/balena-io-modules/balena-multibuild/pull/96

I've tested `balena build` with single and multi-container projects (including `balena-node-hello-world`), :-) single-stage and multi-stage Dockerfiles, the latest Docker Desktop on macOS (Docker Desktop v4.0.1, Docker Engine v20.10.8, with `binfmt_misc` support), and also with balenaEngine on balenaOS (no `binfmt_misc` support).

Things I have not tested:
* I have not tested whether `FROM --platform` breaks anything in comparison to the existing support for `FROM --platform`.
* I have not tested `balena push` nor `balena deploy`.

Reply:
thanks for your help @pdcastro ... this is definitely a team effort.  One thing that I still need to test (just realized) is a third party registry.  

Sanity checks
- [ ] balena build with single and multi-container projects (including balena-node-hello-world) :-)
- [ ] Single-stage and multi-stage Dockerfiles
- [ ] Using a few FROM --platform=linux/arm/v7 lines
- [ ] The latest Docker Desktop on macOS (Docker Desktop v4.0.1, Docker Engine v20.10.8, with binfmt_misc support)
- [ ] balenaEngine on balenaOS (no binfmt_misc support)
- [ ] `balen deploy`
- [ ] `balena push`
- [ ] third party registries

Putting this back in `draft` until we cover those.

Reply:
So, I don't know if I should be shocked, scared, or both.  But the latest changes are passing all tests for me.  

* Build with multi arch v2 manfiest Pi and Nuc
* Build with single arch v2 manifest Pi and Nuc
* Build with single arch V1 manifest Pi and Nuc
* Build with third party registry